config/v1/types_build.go

```diff
@@ -87,6 +91,20 @@ type ProxyConfig struct {
 	NoProxy *string `json:"noProxy,omitempty"`
 }
 
+type RegistriesList []string
+
+type RegistriesConfig struct {
+	// SearchRegistries lists the registries to search for images if an image repository is not specified in an image pull spec
+	// +optional
+	SearchRegistries RegistriesList `json:"searchRegistries,omitempty"`
```

Comment:
This discussion doesn't seem to be resolved:
https://github.com/openshift/api/pull/144#discussion_r241882773

Reply:
My understanding is that we shouldn't have default search repos - if we're in agreement on that then I think we need to doc the behavior change (alongside the new process for updating the search registries).

Reply:
There will be no default search behavior:
https://coreos.slack.com/archives/CE4L0F143/p1547071004002600